servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.h

```diff
@@ -33,12 +33,22 @@
 
 #include "core/templates/paged_allocator.h"
 #include "servers/rendering/renderer_rd/effects/resolve.h"
+#include "servers/rendering/renderer_rd/effects/taa.h"
 #include "servers/rendering/renderer_rd/forward_clustered/scene_shader_forward_clustered.h"
 #include "servers/rendering/renderer_rd/pipeline_cache_rd.h"
 #include "servers/rendering/renderer_rd/renderer_scene_render_rd.h"
 #include "servers/rendering/renderer_rd/shaders/scene_forward_clustered.glsl.gen.h"
 #include "servers/rendering/renderer_rd/storage_rd/utilities.h"
 
+#define RB_SCOPE_CLUSTERED SNAME("clustered")
```

Comment:
Keep in mind we will have forward and deferred clustered eventually and they don't use the same buffers.

Reply:
Made this more explicit: `#define RB_SCOPE_FORWARD_CLUSTERED SNAME("forward_clustered")`